Users/Docker_Image.rst

```diff
@@ -1,6 +1,8 @@
 What is Docker?
 ===============
 
+According to `opensource.com <https://opensource.com/resources/what-docker>`__:
```

Comment:
Hmm that reads bad as a first sentence. Rather I would make some kind of citation at the end^^

Also:
> The layout needs to be very clear that it is an unmodified copy only, and no editing of the text is desirable.

Not sure that this intention gets to the reader here^^

Reply:
Made the requested changes